src/pages/page/page.jsx

```diff
@@ -328,7 +328,7 @@ var Page = React.createClass({
       var elements = {};
 
       page.elements.forEach(element => {
-        var element = this.flatten(element);
+        element = this.flatten(element);
```

Comment:
`element` is already defined so we don't need the `var` (or else the linter will complain and bail).